### PR TITLE
chore(prettier): update prettier

### PR DESCRIPTION
**Pull request by alex-okrushko**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

Run prettier for every commit as a `pre-commit` hook.

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[x] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Closes #

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by brandonroberts** [comment]
Aren't we already doing this here?
 https://github.com/ngrx/platform/blob/master/package.json#L56

**Comment by alex-okrushko** [comment]
> Aren't we already doing this here? https://github.com/ngrx/platform/blob/master/package.json#L56

Didn't know there was a `lint-staged` config. I wasn't familiar with that lib.
This PR was introducing `pretty-quick`. Let's keep lint-staged, but this PR is also updating `husky` to v8, which changes how hooks are set up: https://typicode.github.io/husky/#/?id=husky-4-to-8-cli

I'll update to use `lint-staged`.

File: package.json [hunk 1]
```diff
@@ -162,7 +162,7 @@
     "express": "^4.16.4",
     "fs-extra": "^10.1.0",
     "glob": "^7.1.2",
-    "husky": "^1.2.0",
+    "husky": "^8.0.1",
```
Comment (timdeschryver) [inline package.json:165]: You also need to add a `prepare` script in order to register the githooks.

https://typicode.github.io/husky/#/?id=manual

```json
{
  "scripts": {
    "prepare": "husky install"
  }
}
```


You should also be able to drop the dependency to `husky` if you add this to the prepare script.

```json
{
  "prepare": "git config core.hookspath .githooks"
}
```

The githook itself then looks like this

```
#!/usr/bin/env sh

npx lint-staged
```

Comment (alex-okrushko) [reply package.json:166]: Yes, you are right.

Let's stick with `husky` for now.

**Review by timdeschryver — APPROVED** [review]

**Review by brandonroberts — APPROVED** [review]

**Pull request merged**